### ISSUE TITLE
RuntimeError: Could not find a number that isn't in use

### ISSUE THREAD

**Issue by xoriole** [Tribler/py-ipv8#333]
From error reporter of Tribler 7.1.0

Platform:	Windows-10-10.0.17134-SP0(AMD64)
Uptime:	58094.0280001
```python
Traceback (most recent call last):
File "TriblerGUI\event_request_manager.py", line 130, in on_read_data
RuntimeError: [Failure instance: Traceback: <type 'exceptions.RuntimeError'>: Could not find a number that isn't in use
lib\site-packages\twisted\internet\base.py:1243:run
lib\site-packages\twisted\internet\base.py:1252:mainLoop
lib\site-packages\twisted\internet\base.py:878:runUntilCurrent
lib\site-packages\twisted\internet\task.py:239:__call__
--- <exception caught here> ---
lib\site-packages\twisted\internet\defer.py:150:maybeDeferred
Tribler\pyipv8\ipv8\dht\discovery.py:181:ping_all
Tribler\pyipv8\ipv8\dht\community.py:184:ping_all
Tribler\pyipv8\ipv8\dht\community.py:191:ping
Tribler\pyipv8\ipv8\dht\community.py:67:__init__
Tribler\pyipv8\ipv8\requestcache.py:53:__init__
Tribler\pyipv8\ipv8\requestcache.py:63:find_unclaimed_identifier
]
```

**Comment by devos50** [comment]
It seems that the DHT is pinging other users are a very fast rate, thus exhausting all 'available' random numbers. We should give a higher priority to this issue since it is an indication that we are sending many requests to other users, at a fast rate.

**Comment by qstokkink** [comment]
Related to #273.

**Comment by qstokkink** [comment]
@egbertbouman was this fixed by #335?

**Comment by egbertbouman** [comment]
@qstokkink It should be since there is now less traffic, and messages no longer share the same request-cache prefix. Tribler is running stable now on my machine for >38 hours. I'll reopen if it reoccurs.
